src/components/elements/TableFilters/Inputs/FilterDropDown.tsx

```diff
@@ -34,21 +34,21 @@ const FilterDropDown = (props: PropsWithChildren<FilterDropDownProps>) => {
       {({ open, value }) => (
         <div className="flex items-center gap-3">
           <When condition={!!props.label}>
-            <Listbox.Label as={Text} variant="text-bold-subtitle-500">
+            <Listbox.Label as={Text} variant="text-16-bold">
               {props.label}
```

Comment:
Does this need to be wrapped in `t`? Please check the consumers of this component and see if the label already being passed in is translated.

Reply:
Thank you for pointing that out. After reviewing, I confirmed that the label is not being passed translated, so I wrapped the label in t to ensure translation.